app/views/layouts/application.html.erb

```diff
@@ -1,13 +1,22 @@
 <!DOCTYPE html>
 <html>
-  <head>
+  <head lang="en">
     <title><%= full_title(yield(:title)) %></title>
     <%= csrf_meta_tags %>
-    <%= stylesheet_link_tag    'application', media: 'all',
-                                              'data-turbolinks-track': 'reload' %>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <meta name="author" content="Patricia Ciortea">
+    <link rel="icon" href="https://cdn.onlinewebfonts.com/svg/img_255075.png" type="image/x-icon">
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.1.0/css/bootstrap.min.css">
```

Comment:
you do not need this line if you have the bootstrap gem imported in your application